android/app/build.gradle.kts

```diff
@@ -59,7 +73,7 @@ android {
             )
         }
         getByName("debug") {
-            signingConfig = signingConfigs.getByName("debug")
+            // Debug builds don't need signing config for development
```

Comment:
@cogwheel0 check this if you want to keep it as is, I was unable to run as I didn't have any signing config.